deploy/compose/manifests/apply_manifests.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env sh
+# shellcheck disable=SC2039,SC2155,SC2086
```

Comment:
I feel like it'd be better to change to bash and also *not* disable these script wide.

Going with bash removes the need for 2039.
2155 is important to not ignore script wide
dittor for 2086

Reply:
copy that. updated